src/internal/operations/backup_test.go

```diff
@@ -1005,13 +1041,13 @@ func (suite *BackupOpSuite) TestBackupOperation_MergeBackupDetails_AddsItems() {
 			},
 			inputMans: []*kopia.ManifestEntry{
 				{
-					Manifest: makeManifest(backup1.ID, ""),
+					Manifest: makeManifest(suite.T(), backup1.ID, ""),
 					Reasons: []kopia.Reason{
 						pathReason1,
 					},
 				},
 				{
-					Manifest: makeManifest(backup2.ID, ""),
+					Manifest: makeManifest(suite.T(), backup2.ID, ""),
```

Comment:
May want to consider refactoring `makeManifest` into a few constants as it is reused several times.

Reply:
This code is not part of the current PR.